tests/blockchain/eth/contracts/integration/test_intercontract_integration.py

```diff
@@ -675,7 +823,7 @@ def test_all(testerchain, token, escrow, policy_manager, adjudicator, user_escro
     tokens_amount = escrow.functions.minerInfo(user_escrow_1.address).call()[VALUE_FIELD]
     tx = user_escrow_proxy_1.functions.withdrawAsMiner(tokens_amount).transact({'from': ursula3})
     testerchain.wait_for_receipt(tx)
-    assert 10000 < token.functions.balanceOf(ursula1).call()
+    assert 9370 < token.functions.balanceOf(ursula1).call()
```

Comment:
We should clarify how these amounts (630, 930, 9370, etc.) were computed.

Reply:
✔️